### DISCUSSION TITLE
Badges not updating when requirements met on private business organisation repo

### DISCUSSION THREAD

**Discussion by LiamDHall** [community/community#78975] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I should have more achievements tan displayed on my profile.

All of my badges are before our company turned into a business account.

**Comment by erickdc7** [comment]
There are a few possible reasons why your badges are not updating:

- **Your badges are based on public activity only**. Some badges, such as the Arctic Code Vault Contributor badge or the Mars 2020 Helicopter Contributor badge, are only awarded based on public contributions to qualifying repositories. If your company's repositories are private, they will not count towards these badges.
- **Your badges are based on specific time periods**. Some badges, such as the Hacktoberfest badge or the GitHub Sponsors badge, are only awarded based on activity within a certain time frame. If you met the requirements for these badges before or after the specified period, they will not show up on your profile.
- **Your badges are based on specific events or programs**. Some badges, such as the GitHub Campus Expert badge or the GitHub Star badge, are only awarded based on participation in certain events or programs. If you did not apply or qualify for these events or programs, you will not receive these badges.

If none of these reasons apply to your situation, you can try the following steps to troubleshoot your badge issue:

- **Refresh your profile page**. Sometimes, it may take a while for your badges to update on your profile. You can try refreshing your profile page or clearing your browser cache to see if your badges appear.
- **Check your email settings**. Some badges, such as the GitHub Sponsors badge, require you to verify your email address before they show up on your profile. You can check your email settings and make sure you have a verified email address associated with your GitHub account.
- **Contact GitHub Support**. If none of the above steps work, you can contact GitHub Support and provide them with the details of your badge issue. They may be able to help you resolve the problem or explain why your badges are not updating.

I hope this helps!

**Comment by github-staff** [comment]
Thanks so much for posting in our Community @LiamDHall. Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀
